flutter_local_notifications/lib/src/platform_specifics/android/notification_details.dart

```diff
@@ -291,4 +292,8 @@ class AndroidNotificationDetails {
   /// For most styles, the colouring will only be applied if the notification is
   /// for a foreground service notification.
   final bool colorized;
+
+  /// Set custom notification count.
+  /// Only displayed in compatible home app.
+  final int? badgeNumber;
```

Comment:
To add to what I wrote above, given that the functionality may not actually show a badge number but may even show the number on the notification itself, my suggestion would be that the example app has a separate Android-specific example for this new feature. Otherwise this may confuse those using the example app where they expect to see a badge number but don't see it because they're using an older version of Android. This could result in the community trying to report a bug

Reply:
I got it.  I'll check and fix 🙇

Reply:
I fixed it. Please let me know if anything is missing.